multigeiger/translations_de.h

```diff
@@ -0,0 +1,52 @@
+/* TR, 07.04.2022
+ add new translation strings to ALL files translations_xx.h !!!
```

Comment:
translations have to work based on the already established workflow we already use for documentation (standard gettext / po files / transifex as nice UI for translators).

do not reinvent the wheel.

and also please do not put something major like this into another pull request.

Reply:
I did not reinvent the wheel, this is all (!) ported from the Luftdaten-Sensor project, with some improvements...
This is an ESP32, not a computer, and you really have to tweak the code for memory and size consumption. You have to move this from the heap and put it into flash memory/PROGMEM as much as possible, to get it performing at least somehow. Standard gettext & po files will not really work with an ESP32. I'm not even sure it the macros for po/gettext are available for the ESP platform ( I didn't check this, to be honest).
On the other other hand : When else would you need text translations in a web page, if not upon introduction of the pages themselves ...

Reply:
this PR is way to big and unfocussed IMHO.

it needs to get split up, please start with a PR that ONLY implements the http pages (in English).

no translations, no influxdb.